### DISCUSSION TITLE
WiFi button problem in Windows 10

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#3484] [Q&A]
Hello, a few weeks ago I changed the behavior of the wifi button in windows.
What I did was I put it to open the Windows 8 drop-down menu.
The problem is that now I want it to go back to how it was before, but it won't let me, it is still with the design of Windows 8.
Any solution?
![windows8flyout](https://github.com/user-attachments/assets/447b4899-d6bf-4aea-846d-ee670327cf9c)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by pyrates999** [comment]
You can also try restarting explorer.exe using task manger by killing the process and running it again.
